isaric/parsers/ncov-france.toml

```diff
@@ -466,14 +466,35 @@
   is_present = { field = "signes_clin___6", values = { 1 = true } }
 
 [[observation]]
-  name = "oxygen_flow_volume_max"
+  name = "oxygen_flow_rate"
   phase = "admission"
   date = { ref = "admissionDate" }
+
   [observation.value]
-    combinedType = "max"
+    combinedType = "firstNonNull"
     fields = [
       { field = "a_o2_on", description = "O2 flow (L/min), nasal O2" },
       { field = "a_of_o2", description = "O2 flow (L/min), HFNC" },
+      { field = "a_vni_ti", description = "Inspiratory trigger (L/min)" },
```

Comment:
I'm not sure inspiratory trigger can be used for oxygen flow, seems like a setting on the NIV equipment